lib/active_interaction/base.rb

```diff
@@ -105,6 +119,9 @@ def method_missing(*args, &block)
       # @param options [Hash]
       def add_filter(klass, name, options, &block)
         raise InvalidFilterError, %("#{name}" is a reserved name) if Inputs.reserved?(name)
+        if (invalid_options = options.keys - ALLOWED_KWARGS).any?
+          raise InvalidFilterError, "invalid options: #{invalid_options.join(', ')}"
```

Comment:
Is the right exception to use? Also I wanted to make sure the options were in the message to aid in debugging but not sure if the format should change.